language-configuration.json

```diff
@@ -1,75 +1,43 @@
 {
-	"comments": {
-		"lineComment": "#",
-		"blockComment": [
-			"/*",
-			"*/"
-		]
-	},
-	"brackets": [
-		[
-			"{",
-			"}"
-		],
-		[
-			"[",
-			"]"
-		],
-		[
-			"(",
-			")"
-		]
-	],
-	"autoClosingPairs": [
-		{
-			"open": "{",
-			"close": "}"
-		},
-		{
-			"open": "[",
-			"close": "]"
-		},
-		{
-			"open": "(",
-			"close": ")"
-		},
-		{
-			"open": "\"",
-			"close": "\"",
-			"notIn": [
-				"string"
-			]
-		},
-		{
-			"open": "'",
-			"close": "'",
-			"notIn": [
-				"string",
-				"comment"
-			]
-		}
-	],
-	"autoCloseBefore": ";:.,=}])> \n\t\"",
-	"surroundingPairs": [
-		[
-			"{",
-			"}"
-		],
-		[
-			"[",
-			"]"
-		],
-		[
-			"(",
-			")"
-		],
-		[
-			"\"",
-			"\""
-		],
-		[
-			"'",
-			"'"
-		]
-	]
-}
\ No newline at end of file
+  "comments": {
```

Comment:
Beware formatting this file. This is used if there is no LS present, and used before the LS when present. I've had trouble in the past with subtle changes in this file causing syntax highlighting issues in the past. One of the reasons I moved to defining this in Typescript instead of json. We should test it in a packaged VSIX[1] client side with our set of example terraform projects.

I would be more comfortable if this file was formatted and tested outside of this PR. 

[1] This file is read client side, thus the need for a package test

Reply:
Very good to know, thanks.

I've excluded this file